addons/binding/org.openhab.binding.mihome/ESH-INF/thing/sensor_switch.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mihome" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="sensor_switch">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+
+		<label>Xiaomi Mi Wireless Switch</label>
+		<description>Round-shaped mini wireless switch that allows to trigger four types of event: single click, double click, long click press, long click release.</description>
+
+		<channels>
+			<channel id="button" typeId="button" />
+			<channel id="batteryLevel" typeId="system.battery-level" />
+			<channel id="lowBattery" typeId="system.low-battery" />
+		</channels>
+
+		<config-description>
+			<parameter name="itemId" type="text" readOnly="true">
+				<label>Item ID</label>
+				<description>An identifier for the item connected.</description>
+				<required>true</required>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="button">
```

Comment:
> So why not adding this to the SYSTEM_BUTTON?

Because I haven't seen this on any other device/system. As said above: It imho is a weird mix of "raw" and "higher-level". I fully agree that raw PRESSED/RELEASED events are cool - but this is why we the SYSTEM_RAWBUTTON. Any sensible system would send the PRESSED event in the moment the button is pressed and not a few hundred ms later when it decides for itself that the press does not seem to be a short or double click...